### PR TITLE
Add 'SHOW realtions' method to console

### PR DESCRIPTION
**Pull request by EinKrebs**

### CODE REVIEW

**Comment by EinKrebs** [comment]
Resolves #512

File: test/feature/features/memqdb.feature [hunk 1]
```diff
@@ -10,12 +10,6 @@ Feature: MemQDB save state into a file
     """
     CREATE DISTRIBUTION ds1 COLUMN TYPES integer;
     CREATE DISTRIBUTION ds2 COLUMN TYPES varchar;
-    ADD KEY RANGE krid1 FROM 1 TO 10 ROUTE TO sh1 FOR DISTRIBUTION ds1;
```
Comment (reshke) [inline test/feature/features/memqdb.feature:13]: why?

File: test/regress/tests/console/expected/show_relations.out [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+SHOW relations WHERE distribution_id = 'ds1';
+ Relation name | Distribution ID |  Distribution key  
+---------------+-----------------+--------------------
+ a             | ds1             | ("a_id", identity)
+ b             | ds1             | ("b_id", murmur)
+(2 rows)
+
+SHOW relations WHERE unknown_param = 'ds1';
+ERROR:  column unknown_param not exists.
```
Comment (Denchick) [inline test/regress/tests/console/expected/show_relations.out:55]: `ERROR:  column unknown_param does not exist.`

File: pkg/clientinteractor/interactor.go [hunk 3]
```diff
@@ -868,3 +870,60 @@ func (pi *PSQLInteractor) BackendConnections(ctx context.Context, shs []shard.Sh
...
+		TextOidFD("Relation name"),
+		TextOidFD("Distribution ID"),
+		TextOidFD("Distribution key"),
+	}}); err != nil {
+		spqrlog.Zero.Error().Err(err).Msg("")
+		return err
+	}
+
+	dss := make([]string, len(dsToRels))
+	i := 0
```
Comment (Denchick) [inline pkg/clientinteractor/interactor.go:887]: You don't need this

**Review by Denchick — CHANGES_REQUESTED** [review]

**Review by Denchick — APPROVED** [review]

**Pull request merged**